docs/redux/redux-async.md

```diff
@@ -55,30 +55,25 @@ render(
 Store is configured to accept thunk middleware
 
 ```js
-import {createStore, applyMiddleware} from 'redux';
+import {configureStore, createSlice} from '@reduxjs/toolkit';
 import thunkMiddleware from 'redux-thunk';
-import systemSettingsReducer from '../reducers';
+import rootSlice from '../reducers';
 
-export default function configureStore (initialState) {
-	const store = createStore(
-		systemSettingsReducer,
+
+
+export default function configureAppStore (initialState) {
+	return configureStore({
+		reducer: rootSlice.reducer,
 		initialState,
-		applyMiddleware(thunkMiddleware) // lets us dispatch functions
-	);
-	return store;
+		middleware: [thunkMiddleware]
```

Comment:
As we read from the redux-toolkit docs, https://redux-toolkit.js.org/introduction/getting-started#whats-included, `redux-thunk` is included by default. Would you please update the documentation based on this information as well as the code?

Reply:
https://redux-toolkit.js.org/api/configureStore#middleware we had `applyMiddleware` in the older code, so I had to provide the middleware we used before using the new syntax, passing it to the `middleware` field in the `configureStore`